tests/cases/struct/LV_NeoHookean_passive/svFSI.inp

```diff
@@ -0,0 +1,90 @@
+#----------------------------------------------------------------
+# General simulation parameters
+
+Continue previous simulation: f
+Number of spatial dimensions: 3
+Number of time steps: 100
```

Comment:
Make sure that all `.inp` and `.xml` files are perfectly matching. For instance, in this case the number of time steps is different between Fortran and C++

Reply:
Good catch thanks

Reply:
Following in person discussion with Matteo and Martin, removing all svFSI.inp files from test cases